powerdns/models/powerdns.py

```diff
@@ -627,9 +622,20 @@ def update_serial(sender, instance, **kwargs):
         soa.save()
 
 
+@receiver(post_save, sender=Domain, dispatch_uid='domain_update_ptr')
+def update_ptr(sender, instance, **kwargs):
```

Comment:
So you propose to limit calling this fn only for specific action on domain?
The action is change on domain.auto_ptr field?